maven-resolver-transport-http/src/main/java/org/eclipse/aether/transport/http/HttpTransporter.java

```
@@ -691,8 +690,7 @@ public void handle(CloseableHttpResponse response) throws IOException, TransferC
                 if (lastModifiedHeader != null) {
                     Date lastModified = DateUtils.parseDate(lastModifiedHeader.getValue());
                     if (lastModified != null) {
-                        Files.setLastModifiedTime(
-                                task.getDataFile().toPath(), FileTime.fromMillis(lastModified.getTime()));
+                        task.getDataFile().setLastModified(lastModified.getTime());
```

This will hide the actual exception. I'd move the exception to trace level

---

I disagree, as this is the _only_ place where `Files` is used, while in _many other places_ (see master PR, installer, deployer, etc) `File.setLastModifiedTime` is used instead. IMHO this PR just "aligns" this one cuckoo egg with the rest.

---

While using path or file is something I don't care debating I think tracing there was an error and therefore the lastmodified value is wrong is valuable in the log so please log it (`if (!setlastmodified) log.trace()`).

---

@rmannibucau As I told to @michael-o , I find this proposal futile, and here is why: user @Jurrie spotted this problem ONLY HERE as this was the only one spot using method that throws (`Files.setLastModified`). If you look at master PR https://github.com/apache/maven-resolver/pull/468 it will show you all the places where time mtime is set (as master was ported from `File` to `Path`). So all those place will not log anything and are "forgiving" when they try but fail to set mtime, and hence, logging it only here makes no sense IMHO. If you want, you can make a PR for 1.9.x that sweep changes all `File.setLastModified` usages and logs something at TRACE, but I find it useless change (as it never logged that so far).

---

For master I agree, will prep a followup PR that does log